analysis/active_analyses.R

```diff
@@ -224,102 +214,102 @@ for (c in cohorts) {
-    # 
-    # df[nrow(df)+1,] <- c(cohort = c,
-    #                      exposure = exposure, 
-    #                      outcome = i,
-    #                      ipw = ipw, 
-    #                      strata = strata,
-    #                      covariate_sex = covariate_sex,
-    #                      covariate_age = covariate_age,
-    #                      covariate_other = "cov_cat_ethnicity;cov_cat_deprivation;cov_cat_smoking_status;cov_bin_carehome_status;cov_num_consulation_rate;cov_bin_healthcare_worker;cov_bin_liver_disease;cov_bin_ckd;cov_bin_cancer;cov_bin_hypertension;cov_bin_diabetes;cov_bin_obesity;cov_bin_copd;cov_bin_ami;cov_bin_isch;cov_bin_history_cog_imp_sympt;cov_bin_history_mnd;cov_bin_history_migraine;cov_bin_history_any_dementia;cov_bin_history_parkinson;cov_bin_history_ms;cov_bin_history_parkinson_risk",
-    #                      cox_start = cox_start,
-    #                      cox_stop = cox_stop,
-    #                      study_start = ifelse(c=="prevax", "2020-01-01", "2021-06-01"),
-    #                      study_stop = ifelse(c=="prevax", "2021-12-14", "2021-12-14"), #original study stop: 2021-06-18
-    #                      cut_points = ifelse(c=="prevax", "1;28;197;365;714", "1;28;197"), #original cut points: 28;197;365
-    #                      controls_per_case = controls_per_case,
-    #                      total_event_threshold = total_event_threshold,
-    #                      episode_event_threshold = episode_event_threshold,
-    #                      covariate_threshold = covariate_threshold,
-    #                      age_spline = FALSE,
-    #                      analysis = "sub_age_80_110")
+    df[nrow(df)+1,] <- c(cohort = c,
+                         exposure = exposure,
+                         outcome = i,
+                         ipw = ipw,
+                         strata = strata,
+                         covariate_sex = covariate_sex,
+                         covariate_age = covariate_age,
+                         covariate_other = all_covars,
+                         cox_start = cox_start,
+                         cox_stop = cox_stop,
+                         study_start = ifelse(c=="prevax", "2020-01-01", "2021-06-01"),
+                         study_stop = ifelse(c=="prevax", "2021-12-14", "2021-12-14"), #original study stop: 2021-06-18
+                         cut_points = ifelse(c=="prevax", "1;28;197;365;714", "1;28;197"), #original cut points: 28;197;365
+                         controls_per_case = controls_per_case,
+                         total_event_threshold = total_event_threshold,
+                         episode_event_threshold = episode_event_threshold,
+                         covariate_threshold = covariate_threshold,
+                         age_spline = FALSE,
+                         analysis = "sub_age_18_39")
+    
+    ## analysis: sub_age_40_65 ------------------------------------------------
```

Comment:
65 is not included in this age category, please rename.

Reply:
Done.